packages/graphback-runtime/src/service/CRUDService.ts

```diff
@@ -11,10 +10,7 @@ export interface CRUDServiceConfig {
    * PubSub implementation for creating subscriptions
    */
   pubSub?: PubSubEngine
-  /**
-   * Optional logging service
-   */
-  logger?: GraphbackMessageLogger
```

Comment:
Hmm - I could see it as valuable to have but disabled by default, so the logger would be set but it would have an `enabled` flag passed to the constructor

Reply:
hmm.. general we need to have platform level logger in core
having separate logger for the each component line service, data provider, plugin etc. will be really really bad API. We have issue to improve logging as whole (thx @machi1990) and we will action it there. This was more like hotfix as extensive logging in node.js is problem (especially console.log)

Reply:
Yes we have this issue https://github.com/aerogear/graphback/issues/1234 to address that. And I could see the logger being configured by the user (to make it custom) so that we do not bring our own logger lib.